src/docs/guides/assets-manager/README.md

```diff
@@ -6,7 +6,7 @@ Based on [Concurrently](https://github.com/open-cli-tools/concurrently) the Orch
 
 Old assets are not compiled as ES6 modules so they don't need these bundlers. For that matter we kept the old `gulpfile.js` which will be now triggered by `Concurrently` when doing `yarn build -gr`.
 
-Concurrently uses an `Assets.json` file that defines actions to execute. The old Gulp pipline uses a `GulpAssets.json` file that defines actions to execute.
+Concurrently uses an `Assets.json` file that defines actions to execute. The old Gulp pipeline uses a `GulpAssets.json` file that defines actions to execute.
```

Comment:
If I'm know wrong @Piedone submitted a PR for fix such issues, right?

Reply:
Not yet, just opened https://github.com/OrchardCMS/OrchardCore/issues/17494.